README.md

```diff
@@ -1,463 +1,161 @@
-![Build status](https://kantord.semaphoreci.com/badges/LibreLingo.svg?style=shields "Build status")
```

Comment:
Certainly the header of the readme looks a lot cleaner now but it also has a lot of empty space and little useful information on what this project is about. 

the current readme doesn't do an amazing job at this either, but right now this makes it a little bit worse.

I would say we should prioritize information that gives a very clear idea to people what the project is about in the header of the readme

Reply:
We can always modify that later, but should shift from what we have right now as it's a bit messy